src/computability/DFA.lean

```diff
@@ -128,3 +128,38 @@ begin
 end
 
 end DFA
+
+namespace language
+
+/-- A language `is_regular` if it is the strings accepted by some DFA -/
+structure is_regular (l : language α) :=
+(σ : Type v)
```

Comment:
This universe polymorphism is likely to cause you unnecessary pain - it means that "regular in universe u" and "regular in universe u+1" are different statements. Since all fintypes are isomorphic to a type in `Type`, the statements are all equivalent so you may as well just quantify over `Type` or just `fin n`